utils/RunONNXModel.py

```diff
@@ -639,6 +647,25 @@ def __init__(self, model_file, **kwargs):
         # e.g. model.so, model.constants.bin, ...
         self.default_model_name = "model"
 
+        # Handle cache_model.
+        if args.cache_model:
+            shared_lib_path = args.cache_model + f"/{self.default_model_name}.so"
+            if not os.path.exists(shared_lib_path):
+                print(
+                    'Cached compiled model not found in "'
+                    + args.cache_model
+                    + '": save model this run.'
+                )
+                args.save_model = args.cache_model
+            else:
+                print(
+                    'Cached compiled model found in "'
+                    + args.cache_model
+                    + '": load model this run.'
+                )
+                args.load_model = args.cache_model
+            args.cache_model = None
```

Comment:
Got it. It is a combination of `save-model` and `load-model` so that we don't need to run twice.